### PR TITLE
🎨 e2e: move widget id from nodeTreeItem to nodeTreeItem's label

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

During the e2e testing it can happen that when puppeteer clicks on the node tree item on the primary column, the fullscreen button gets clicked, bringing an unexpected scenario. This happens when the fullscreen button gets rendered right in the middle of the widget. 

This PR moves the id given to the widget (the one puppeteer looks for) from the entry in the tree to the label in the entry in the tree.

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/component/widget/NodeTreeItem.js [hunk 1]
```diff
@@ -300,6 +300,7 @@ qx.Class.define("osparc.component.widget.NodeTreeItem", {
       // The label
       this.addLabel();
       const label = this.getChildControl("label");
+      osparc.utils.Utils.setIdToWidget(label, "nodeTreeItem");
```
Comment (colinRawlings) [inline services/static-webserver/client/source/class/osparc/component/widget/NodeTreeItem.js:303]: So this rather labels the text of the node?

Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/component/widget/NodeTreeItem.js:303]: kenau

Comment (colinRawlings) [reply services/static-webserver/client/source/class/osparc/component/widget/NodeTreeItem.js:303]: Ah wait now I bothered to read the mr text

**Review by sanderegg — APPROVED** [review]
thanks!

**Pull request merged**